hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILEPanda.py

```diff
@@ -181,3 +184,36 @@ def __init__(self, **kwargs: Any) -> None:
 class PandaHistoSSLMIL(DeepSMILEPanda):
     def __init__(self, **kwargs: Any) -> None:
         super().__init__(encoder_type=HistoSSLEncoder.__name__, **kwargs)
+
+
+class SubPandaImageNetMIL(PandaImageNetMIL):
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        root = "~/workspace/repos/hi-ml/hi-ml-histopathology/src/histopathology/configs/classification"
+        self.crossval_count = 1
+        self.train_csv = os.path.join(root, "custom_subset/panda/sub_train_tiles.csv")
+        self.val_csv = os.path.join(root, "custom_subset/panda/sub_val_tiles.csv")
+
+    def get_data_module(self) -> SubPandaTilesDataModule:
+        image_key = PandaTilesDataset.IMAGE_COLUMN
+        if self.is_finetune:
+            transform = Compose([LoadTilesBatchd(image_key, progress=True)])
+        else:
+            transform = Compose(
+                [
+                    LoadTilesBatchd(image_key, progress=True),
+                    EncodeTilesBatchd(image_key, self.encoder, chunk_size=self.encoding_chunk_size),
+                ]
+            )
+
+        return SubPandaTilesDataModule(
+            train_csv=self.train_csv,
+            val_csv=self.val_csv,
+            root_path=self.local_datasets[0],
+            max_bag_size=self.max_bag_size,
+            batch_size=self.batch_size,
+            transform=transform,
+            cache_mode=self.cache_mode,
+            precache_location=self.precache_location,
+            cache_dir=self.cache_dir,
+        )
```

Comment:
This was mainly to create the baseline for the PANDA subset but apparently this might be useful for testing as well. It might be better to move to a dedicated module config. As for the finetune option, I don't really know if we need it for the benchmark, I kept the exact same logic as in the base config and only change the subset datamodule.

Reply:
Agree that moving to a separate module, maybe in the test folder, could help.